apps/fiona/fiona.cpp

```diff
@@ -81,9 +81,15 @@
     // Thus, we do not enable it in this case.
         #define _GLIBCXX_PARALLEL
     #endif  // #if !defined(__MINGW32__) || defined(__amd64__) || defined(__x86_64__) || defined(__ia64__)
+
+    #if defined(__INTEL_COMPILER)
+        #pragma message("The Intel Compiler crashes with _GLIBCXX_PARALLEL defined (at least until v16.0.2). Therefore, the feature will be disabled.")
+        #if __INTEL_COMPILER < 1600 || (__INTEL_COMPILER == 1600 && __INTEL_COMPILER_UPDATE <= 2)
+            #undef _GLIBCXX_PARALLEL
+        #endif
+    #endif // #if defined(__INTEL_COMPILER)
 #else
     #warning "Please enable OpenMP."
-    #define omp_get_wtime() 0
```

Comment:
This macro and the stub method in /include/seqan/parallel/parallel_macros.h
for omp_get_wtime have a name conflict.

So there is a stub method for the case that omp is not available